DuckDuckGo/TabViewController.swift

```diff
@@ -2345,7 +2345,7 @@ extension TabViewController: SecureVaultManagerDelegate {
     }
     
     func secureVaultManagerIsEnabledStatus(_: SecureVaultManager) -> Bool {
-        let isEnabled = featureFlagger.isFeatureOn(.autofill)
+        let isEnabled = featureFlagger.isFeatureOn(.credentialsSaving)
```

Comment:
This function `secureVaultManagerIsEnabledStatus` generally decides when to provide autofillable values to the webview, should the `credentialsAutofill` subfeature be used here instead?